doc/fastcat_device_config_parameters.md

```diff
@@ -231,16 +236,85 @@ Engineering Units (EU) are radians for revolute actuators and meters for linear
       high_pos_cmd_limit_eu:         1e15
       high_pos_cal_limit_eu:         1e15
       holding_duration_sec:          5.0
-      egd_brake_engage_msec:         10 
-      egd_brake_disengage_msec:      10
-      egd_crc:                       -3260
-      egd_drive_max_current_limit:   10
+      elmo_brake_engage_msec:         10 
+      elmo_brake_disengage_msec:      10
+      elmo_crc:                       -3260
+      elmo_drive_max_current_limit:   10
       smooth_factor:                 0
 ```
 
 
 
-## Egd (Elmo Gold Drive) TODO
+## Egd (Elmo Gold Drive) 
+
+This is a thin wrapper around the JSD EGD device. This does not have fastcat-side profiling nor any notion of gear ratio so the drive must be commanded in encoder counts.
+
+| Parameter                        | Description                                                  |
+| -------------------------------- | ------------------------------------------------------------ |
+| `cs_cmd_freq_hz`                 | The Target loop rate to command the drive in {CSP, CSV, CST} modes |
+| `drive_cmd_mode`                 | Either `CS` or `PROFILED` see notes below for more details |
+| `max_motor_speed`                | Maximum speed this drive may be commanded in counts/sec   |
+| `torque_slope`                   | Rate to apply torque in certain profiled torque command modes |
+| `max_profile_accel`              | ELMO-side profiler acceleration in counts/sec |
+| `max_profile_decel`              | ELMO-side profiler deceleration in counts/sec |
+| `velocity_tracking_error`        | ELMO-side velocity tracking error in counts/sec |
+| `position_tracking_error`        | ELMO-side position tracking error in counts |
+| `peak_current_limit`             | Peak instantaneous current in Amp |
+| `peak_current_time`              | Max apply duration of Peak current before dropping down to Max Continuous current |
+| `continuous_current_limit`       | Max continuously supplied current permitted to actuator |
+| `motor_stuck_current_level_pct`  | See Elmo docs for details on this feature, `0` disables|
```

Comment:
Actually, a value less than 2 disables the protection. Also, setting this parameter alone to less than 2 is sufficient to disable the protection.